tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/LogstashInstallation.java

```diff
@@ -161,7 +166,7 @@ private static void pipeRepeatedly(final File input, final OutputStream out,
             final int count) throws IOException {
             for (int i = 0; i < count; ++i) {
                 try (final InputStream file = new FileInputStream(input)) {
-                    IOUtils.copy(file, out, 16 * 4096);
+                    IOUtils.copy(file, out, 4096 * 16);
```

Comment:
just curious... Is this a generally preferred style and why ? _(I have never had this come up before, and never gave it any thought before now)_.